tests/run_make_tests.pl

```diff
@@ -53,17 +53,7 @@
 $pure_log = undef;
 
 # The location of the GNU make source directory
-$srcdir = undef;
-$fqsrcdir = undef;
-$srcvol = undef;
-
-# The location of the build directory
-$blddir = undef;
-$fqblddir = undef;
-$bldvol = undef;
-
-$make_path = undef;
-@make_command = ();
```

Comment:
Is it not recommended/necessary to init variables?

Reply:
I didn't notice this and was coming from the standpoint of what is in make 4.3 which has changed since 4.2.1 and what was in remake 4.3. 

However since you mention I looked at this. It appears these variables aren't used. So I guess that's whay Paul Smith removed them.